.gitignore

```diff
@@ -4,4 +4,27 @@ node_modules/
 dist/
 .env
 .nyc_output/
-/coverage
\ No newline at end of file
+/coverage
```

Comment:
we don't need any files from `/coverage` folder to be uploaded to this repo.

Please remove this change.